src/_includes/hubspot_cta.liquid

```diff
@@ -0,0 +1,19 @@
+{%- if site.lang == 'es' -%}
+  <span class="{{ include.class }}" id="hs-cta-wrapper-{{ include.hubspot-id }}">
```

Comment:
Now that we're not using the HubSpot classNames can we clean up our own CSS, e.g. remove some of the overrides and importants?

Reply:
I've updated this here: [6874364](https://github.com/codurance/site/commit/68743644414cefda9b9fa0facf098eb3b7abc32c)